web/docs/use/detect/match-threat-intel.md

```diff
@@ -49,35 +49,35 @@ Working with matchers involves three separate steps:
 2. [Add](#add-indicators)/[remove](#remove-indicators) indicators
 3. [Attach to the matcher](#attach-to-matchers) to consume sightings
 
-VAST uniquely identifies matchers by their name, either as specified in the YAML
-configuration or on the command line. Whenever interacting with a matcher, you
-need to pass the name as argument to all operations. The general pattern looks
-as follows:
+Tenzir uniquely identifies matchers by their name, either as specified in the
+YAML configuration or on the command line. Whenever interacting with a matcher,
+you need to pass the name as argument to all operations. The general pattern
+looks as follows:
 
 ```bash
-vast matcher <command> [options] <name>
+tenzirctl matcher <command> [options] <name>
 ```
 
-VAST also supports executing operations on multiple matchers at once, e.g., to
+Tenzir also supports executing operations on multiple matchers at once, e.g., to
 a add an indicator to a many matchers. To this end, simply use a comma-separated
-list for the positional `name` argument, e.g., `vast matcher add a,b,c ...` to
-act on matchers `a`, `b`, and `c`.
+list for the positional `name` argument, e.g., `tenzirctl matcher add a,b,c ...`
+to act on matchers `a`, `b`, and `c`.
 
 :::note Requirements
-To use matchers, make sure that your VAST distribution has the `matcher` plugin
-available, e.g., by checking the output of `vast version`:
+To use matchers, make sure that your Tenzir distribution has the `matcher`
+plugin available, e.g., by checking the output of `tenzir version`:
 
 ```bash
-vast -q --plugins=all version | jq .plugins.matcher
+tenzir -q --plugins=all version | jq .plugins.matcher
```

Comment:
The `jq` here doesn't match the output of the version operator at all.

Reply:
Noted down for a second pass.